app/assets/stylesheets/frontend/views/attachments/_csv-preview.scss

```diff
@@ -0,0 +1,15 @@
+.csv-preview {
+  border: govuk-spacing(1) solid govuk-colour("light-grey");
+  margin-bottom: govuk-spacing(6);
+}
+
+.csv-preview__inner {
+  border: 1px solid govuk-colour("mid-grey");
+  padding: govuk-spacing(2) govuk-spacing(4) 0;
+  overflow: auto;
+
+  &:focus {
+    border: 3px solid govuk-colour("black");
```

Comment:
could `@include govuk-focused-text;`be used?

Reply:
Not quite. Looks like this:
![Screenshot 2021-04-13 at 10 19 45](https://user-images.githubusercontent.com/64783893/114529405-d1b46580-9c41-11eb-8a49-be0e679c9500.png)
I think the focus text include is intended for text elements and the like. I feel like it's a bit too intense for this use case.

For what it's worth, I copied this implementation from the design system docs website on their code blocks.